spec/services/exports/case_log_export_service_spec.rb

```diff
@@ -2,38 +2,77 @@
 
 RSpec.describe Exports::CaseLogExportService do
   let(:storage_service) { instance_double(StorageService) }
+
   let(:export_filepath) { "spec/fixtures/exports/case_logs.xml" }
   let(:export_file) { File.open(export_filepath, "r:UTF-8") }
-  let(:expected_filename) { "core_2022_02_08/dat_core_2022_02_08_0001.xml" }
+
+  let(:expected_data_filename) { "core_2022_02_08/dat_core_2022_02_08_0001.xml" }
+  let(:expected_master_manifest_filename) { "Manifest_2022_02_08_0001.csv" }
```

Comment:
The manifest is a csv and not an xml? :joy:

Reply:
yup :)